test/whitelist.test.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
+const { randomBytes } = require('crypto')
+const { test } = require('tap')
+const { timeout } = require('nonsynchronous')
+const hyperswarm = require('../')
+
+test('whitelist default', async ({ is, fail }) => {
+  const key = randomBytes(32)
+  const swarmA = hyperswarm()
+  const swarmB = hyperswarm()
```

Comment:
can you follow the pattern of the other tests and spin up a bootstrap node etc? this will test against the global dht